src/include/migraphx/tf.hpp

```diff
@@ -7,8 +7,15 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+/// struct to pass in tf options to parser
+struct tf_options
+{
+    bool is_nhwc            = false;
+    unsigned int batch_size = 1;
+};
+
 /// Create a program from a tf pb file (default is nhwc format)
-program parse_tf(const std::string& name, bool is_nhwc);
+program parse_tf(const std::string& name, tf_options options);
```

Comment:
The `options` struct should be optional. It should be `parse_tf(const std::string& name, tf_options options=tf_options{})`